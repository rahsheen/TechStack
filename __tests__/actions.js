import { selectLibrary } from '../src/actions';

it('creates a SELECT_LIBRARY action', () => {
  // You do it
  expect(selectLibrary(1000)).toEqual(
    {
      type: 'select_library',
      payload: 1000
    }
  );
  
  // Jest does it
  expect(selectLibrary(1000)).toMatchSnapshot();
});