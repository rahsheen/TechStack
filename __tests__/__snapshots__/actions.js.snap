// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a SELECT_LIBRARY action 1`] = `
Object {
  "payload": 1000,
  "type": "select_library",
}
`;
